.codecov.yml

```diff
@@ -24,7 +24,7 @@ parsers:
       macro: no
 
 comment:
-  layout: "header, diff"
+  layout: "header, flags, diff"
```

Comment:
this is to distinguish different part of code
https://github.com/pingcap/ticdc/pull/3586#issuecomment-981556744